pkg/apis/pgv2.percona.com/v2/perconapgcluster_types.go

```diff
@@ -79,7 +79,7 @@ type PerconaPGClusterSpec struct {
 	// The major version of PostgreSQL installed in the PostgreSQL image
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Minimum=10
```

Comment:
do we support 10, 11, 12?

Reply:
We support from 12 to 16. If you want, I can fix `Minimum` version.

Reply:
fixed [9858b8d](https://github.com/percona/percona-postgresql-operator/pull/548/commits/9858b8d6096ce99346af6881869a46095e7b1b52)